composer.json

```diff
@@ -16,35 +16,36 @@
 
     "require": {
         "php":">=5.4",
-        "symfony/framework-bundle": "^2.4|^3.0",
-        "symfony/finder": "^2.4|^3.0",
-        "symfony/translation": "^2.4|^3.0",
-        "symfony/asset": "^2.4|^3.0",
-        "symfony/templating": "^2.4|^3.0",
+        "symfony/framework-bundle": "^2.4|^3.0|^4.0",
+        "symfony/finder": "^2.4|^3.0|^4.0",
+        "symfony/translation": "^2.4|^3.0|^4.0",
+        "symfony/asset": "^2.4|^3.0|^4.0",
+        "symfony/templating": "^2.4|^3.0|^4.0",
         "paragonie/random_compat": "^1.1|^2.0"
     },
 
     "require-dev": {
         "amazonwebservices/aws-sdk-for-php": "1.5.*",
         "knplabs/gaufrette": "0.2.*@dev",
-        "oneup/flysystem-bundle": "^1.2",
+        "oneup/flysystem-bundle": "^3.0",
         "phpunit/phpunit": "^4.4",
         "sensio/framework-extra-bundle": "2.*|^3.0",
-        "symfony/browser-kit": "2.*|^3.0",
-        "symfony/class-loader": "2.*|^3.0",
-        "symfony/security-bundle": "2.*|^3.0",
+        "symfony/browser-kit": "2.*|^3.0|^4.0",
+        "symfony/class-loader": "2.*|^3.0|^4.0",
+        "symfony/security-bundle": "2.*|^3.0|^4.0",
         "twistor/flysystem-stream-wrapper": "^1.0"
     },
 
     "suggest": {
         "knplabs/knp-gaufrette-bundle": "0.1.*",
-        "oneup/flysystem-bundle": "^1.2",
+        "oneup/flysystem-bundle": "^3.0",
```

Comment:
Be aware: to keep it compatible with Symfony 2, 3, and 4 as well as PHP 5.6 and 7.x the constraint should be `^1.2|^2.0|^3.0` :)

Reply:
Sorry, didnt' see it's only a suggest. Then you're right!